docs/kernel_providers.rst

```diff
@@ -66,7 +66,7 @@ called *oblong*::
                 'argv': ['oblong-kernel'],
             }
 
-        def make_manager(self, name):
+        def launch(self, name):
             if name == 'standard':
                 return KernelManager(kernel_cmd=['oblong-kernel'],
                                      extra_env={'ROUNDED': '0'})
```

Comment:
This API has changed a bit - `launch()` is now meant to launch the kernel and return `(connection_info, kernel_mgr)`. So the example here should probably show using `SubprocessKernelLauncher`.

(Maybe this points to putting these docs with the JKM repository so it's easier to keep them up to date. But on the other hand, splitting up docs into ever smaller pieces to follow the associated code around feels pretty user-hostile.)

Reply:
Yeah, this was the change that prompted me to say 'wait, let's talk about this'.  

When I was going through the doc, I found myself thinking most of these docs were relative to JKM but figured you "picked" protocol to be the hosting repo.  

I also agree that splitting the docs to equate to their respective repos will be frustrating, but, assuming the split of client into protocol and JKM sticks, I think it's the best approach since users of packages expect associated docs.  We'll want to "link" the two sets of docs more liberally that is usually the case to help "ease their pain" (including links to/from jupyter_client).